File: lista1_js/7.soma_dif.js
//7. Leia 3 números, calcule e escreva a soma dos 2 primeiros e a diferença entre os 2 últimos.
import { question } from "readline-sync"

//Entrada
const num1 = Number(question('Digite o 1º número: '))
const num2 = Number(question('Digite o 2º número: '))
const num3 = Number(question('Digite o 3º número: '))

//Processamento
const soma = num1 + num2
const diferenca = num2 - num3

//Saída
console.log(`\n>>> A soma de ${num1} + ${num2} é ${soma}.`)
console.log(`>>> A soma de ${num2} - ${num3} é ${diferenca}.`)

File: lista1_py/7.soma_dif.py
#7. Leia 3 números, calcule e escreva a soma dos 2 primeiros e a diferença entre os 2 últimos.

#Entrada
num1 = int(input('Digite o 1º número: '))
num2 = int(input('Digite o 2º número: '))
num3 = int(input('Digite o 3º número: '))

#Processamento
soma = num1 + num2
diferenca = num2 - num3

#Saída
print(f'\n>>> A soma de {num1} + {num2} é {soma}.')
print(f'>>> A diferença de {num2} - {num3} é {diferenca}.')